lib/next-amp/src/components/AmpAccess.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import Head from 'next/head';
+import {AmpIncludeCustomElement} from '../../src-gen/';
+
+/**
+ * Renders an amp-access element.
```

Comment:
Should a comment be added here why the default generated source could not be used? Might be interesting to the reader and other people who have to maintain in the future.

Reply:
(And same for all other manually authored components.) That is, have documentation somewhere for maintainers as well as users? But an AMP expert might also be interested - they need to understand the relationship between the React markup and AMP markup as amp.dev documentation talks AMP markup, not React.

Reply:
I'd plan to add this to the readme (or docs).